cfme/tests/cli/test_appliance_console.py

```diff
@@ -1293,8 +1304,58 @@ def test_appliance_console_static_dns():
         caseimportance: low
         caseposneg: negative
         initialEstimate: 1/6h
+        testSteps:
+            1. configure static ipv4
+            2. stop and start EVM service
+            3. reconfigure static ipv4
+            4. configure static ipv6
+            5. stop and start EVM service
+            6. reconfigure static ipv6
+        expectedResults:
+            1.
+            2.  check evm service status while stop and start
+            3. check default secondary DNS is listed or not while entering  DNS
+            4.
+            5. check evm service status while stop and start
+            6. check default secondary DNS is listed or not while entering  DNS
     """
-    pass
+    appliance = temp_appliance_preconfig_long
+    command_set = ("ap", RETURN, "1", "2", RETURN, RETURN, RETURN, RETURN, secondary_dns, RETURN,
+                   TimedCommand("Y", 120))
+    appliance.appliance_console.run_commands(command_set, timeout=30)
+
+    def stop_start_evmserverd(appliance):
+        command_set = ("ap", RETURN, "16", "Y")
+        appliance.appliance_console.run_commands(command_set, timeout=30)
+        wait_for(lambda: appliance.evmserverd.running,
+                 delay=10,
+                 timeout=300,
+                 fail_condition=True,
+                 message='Waiting to stop EVM services',
+                 fail_func=appliance.evmserverd.running
+                 )
+
+        command_set = ("ap", RETURN, "17", "Y")
+        appliance.appliance_console.run_commands(command_set, timeout=30)
+        wait_for(lambda: appliance.evmserverd.running,
+                 delay=10,
+                 timeout=300,
+                 message='Waiting to start EVM services',
+                 fail_func=appliance.evmserverd.running
+                 )
+
+    stop_start_evmserverd(appliance)
+    command_set = ("ap", RETURN, "1", "2", RETURN, RETURN, RETURN, RETURN)
+    result = appliance.appliance_console.run_commands(command_set, timeout=30)
+    assert secondary_dns in result[-1], "Default secondary DNS not found while re-configuring IPV4"
+
+    command_set = ("ap", RETURN, "1", "3", RETURN, RETURN, RETURN, RETURN, RETURN, RETURN,
+                   TimedCommand("Y", 120))
+    appliance.appliance_console.run_commands(command_set, timeout=40)
+    stop_start_evmserverd(appliance)
+    command_set = ("ap", RETURN, "1", "3", RETURN, RETURN, RETURN, RETURN)
+    result = appliance.appliance_console.run_commands(command_set, timeout=30)
+    assert secondary_dns in result[-1], "default Secondary DNS not found while re-configuring IPV6"
```

Comment:
@dgaikwad I still think this portion of the test should be parametrized to avoid to avoid the small code duplication. What are your thoughts on this?